src/lib.rs

```diff
@@ -163,8 +176,24 @@ impl<L: LogLevel> fmt::Display for Verbosity<L> {
     }
 }
 
+impl<L: LogLevel> From<Verbosity<L>> for LevelFilter {
+    fn from(v: Verbosity<L>) -> Self {
+        v.log_level_filter()
+    }
+}
+
+impl<L: LogLevel> From<LevelFilter> for Verbosity<L> {
+    fn from(filter: LevelFilter) -> Self {
+        let default_verbosity = level_value(L::default());
+        let verbosity = level_filter_value(filter);
+        let verbose = verbosity.saturating_sub(default_verbosity);
+        let quiet = default_verbosity.saturating_sub(verbosity);
+        Verbosity::new(verbose, quiet)
+    }
+}
+
 /// Customize the default log-level and associated help
-pub trait LogLevel {
+pub trait LogLevel: Clone {
```

Comment:
Clone is required because of the `#[serde(into = "LevelFilter")]` setting (https://serde.rs/container-attrs.html#into).

This is technically a semver breaking change, as now implementors of LogLevel need to make their implementation implement Clone. I think there may be a way to fix this with another serde attribute.

Reply:
`#[serde(bounds(serialize = "L: Clone")))` does the trick.